mon-pix/app/services/session.js

```diff
@@ -84,7 +84,7 @@ export default class CurrentSessionService extends SessionService {
   async _checkForURLAuthentication(transition) {
     if (transition.to.queryParams && transition.to.queryParams.externalUser) {
```

Comment:
suggestion : supprimer les commentaires et ajouter une constante
`const isNewExternalUser = transition.to.queryParams && transition.to.queryParams.externalUser`
Mettre ça dans l'assertion un test :
`Logout user when a new external user is authenticated without redirecting the use to the login page.`

Reply:
Je rajoutes dans ce cas ta suggestion sur le ticket Jira.